src/components/Table/rowRenderer.js

```diff
@@ -20,19 +20,23 @@ export default function defaultRowRenderer(
     style,
   },
   {
-    openIndex,
+    selectedIndex,
     ExpandedComponent,
     expandedHeight,
     rowHeight,
     expandedRowStyles,
     expandedData,
     tableData,
     theme,
```

Comment:
Since you're using `theme.table` a lot more now, it might clean things up a tiny bit to destructure this out:
```js
theme: {
  table,
  expandedRow
}
```

then you can just write `table.header` (or whatever).